research/one-shot-vfl/README.md

```diff
@@ -1,28 +1,34 @@
-# One-shot VFL
+# One-shot Vertical Federated Learning with CIFAR-10
 
-This directory will contain the code for the methods described in
+This example includes instructions on how to run single-client [one-shot vertical federated learning](https://arxiv.org/abs/2303.16270) using the 
+CIFAR-10 dataset and the [FL simulator](https://nvflare.readthedocs.io/en/latest/user_guide/fl_simulator.html).
 
-### Communication-Efficient Vertical Federated Learning with Limited Overlapping Samples ([arXiv:2303.16270](https://arxiv.org/abs/2303.16270))
+We assume one client holds the images, and the other client holds the labels to compute losses and accuracy metrics. 
+Activations and corresponding gradients are being exchanged between the clients using NVFlare.
 
-###### Abstract:
+<img src="./figs/oneshotVFL.png" alt="One-shot VFL setup" width="800"/>
 
-> Federated learning is a popular collaborative learning approach that enables clients to train a global model without sharing their local data. Vertical federated learning (VFL) deals with scenarios in which the data on clients have different feature spaces but share some overlapping samples. Existing VFL approaches suffer from high communication costs and cannot deal efficiently with limited overlapping samples commonly seen in the real world. We propose a practical vertical federated learning (VFL) framework called **one-shot VFL** that can solve the communication bottleneck and the problem of limited overlapping samples simultaneously based on semi-supervised learning. We also propose **few-shot VFL** to improve the accuracy further with just one more communication round between the server and the clients. In our proposed framework, the clients only need to communicate with the server once or only a few times. We evaluate the proposed VFL framework on both image and tabular datasets. Our methods can improve the accuracy by more than 46.5% and reduce the communication cost by more than 330× compared with state-of-the-art VFL methods when evaluated on CIFAR-10.
+For instructions of how to run CIFAR-10 in real-world deployment settings, 
+see the example on ["Real-world Federated Learning with CIFAR-10"](../../cifar10/cifar10-real-world/README.md).
 
-## License
```

Comment:
We need to keep the License section. Please use the same as [here](https://github.com/NVIDIA/NVFlare/tree/dev/research/auto-fed-rl#license) and also add the same [LICENSE](https://github.com/NVIDIA/NVFlare/blob/dev/research/auto-fed-rl/LICENSE) file to this directory.

Reply:
Done.